Apps/phadldap/adldap.json

```diff
@@ -0,0 +1,1311 @@
+    "app_wizard_version": "1.0.2",
+    "python_version": "3",
+    "latest_tested_versions": [
+        "ldap3 on 1/05/2021"
+    ],
+    "pip_dependencies": {
+        "wheel": [
+            {
+                "module": "ldap3",
+                "input_file": "wheels/ldap3-2.6.1-py2.py3-none-any.whl"
+            },
+            {
+                "module": "pyasn1",
+                "input_file": "wheels/pyasn1-0.4.7-py2.py3-none-any.whl"
+            }
+        ]
+    },
+    "configuration": {
+        "server": {
+            "description": "The Active Directory Server hostname, IP, or VIP for binding",
+            "data_type": "string",
+            "required": true,
+            "value_list": [],
+            "order": 0
+        },
+        "username": {
+            "description": "The username with which to bind to LDAP",
+            "data_type": "string",
+            "required": true,
+            "value_list": [],
+            "order": 1
+        },
+        "password": {
+            "description": "The password for the binding user",
+            "data_type": "password",
+            "required": true,
+            "order": 2
+        },
+        "force_ssl": {
+            "description": "Force the use of SSL protocol. Note that some actions are not possible without secure binding!",
+            "data_type": "boolean",
+            "required": true,
+            "default": true,
+            "order": 3
+        },
+        "validate_ssl_cert": {
+            "description": "Select if you want to validate the LDAP SSL certificate",
+            "data_type": "boolean",
+            "required": true,
+            "default": false,
+            "order": 4
+        },
+        "ssl_port": {
+            "description": "The port to bind for SSL (default 636)",
+            "data_type": "numeric",
+            "required": true,
+            "value_list": [],
+            "default": "636",
+            "order": 5
+        }
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate the asset configuration for connectivity using supplied configuration",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "add group members",
+            "identifier": "add_group_members",
+            "description": "Adds one more more AD objects to one or more groups",
+            "type": "custom",
+            "read_only": false,
+            "versions": "EQ(*)",
+            "parameters": {
+                "use_samaccountname": {
+                    "description": "Specify members AND groups as samaccountnames instead of distinguished names (note: member AND groups will use samaccountname if selected).",
+                    "data_type": "boolean",
+                    "required": false,
```

Comment:
I don't think "required": false is needed anymore?

Reply:
Going to leave it since it does not hurt and provides a placeholder for the future